logstash-core/lib/logstash/java_pipeline.rb

```diff
@@ -472,6 +472,20 @@ def maybe_setup_out_plugins
   def default_logging_keys(other_keys = {})
```

Comment:
I wonder if `default_logging_keys` is necessarily the right place for this - this is logged in 16 different places in this file. Maybe we could use it more selectively (on startup, on failure?)